cypress/fixtures/analysis.json

```diff
@@ -1378,7 +1378,7 @@
         "source": "Source code",
         "target": ["Containerization", "Linux", "OracleJDK to OpenJDK"],
         "customRule": ["corporate-framework-config.windup.xml"],
-        "effort": 2,
+        "effort": 3,
```

Comment:
@kpunwatk did you sync with devs that updated numbers are fine? Difference between these values and UI values can be caused by the bug.

Reply:
Thanks for noticing, @ibragins  I did contact the devs and modified the test cases accordingly. Please, Review Again, Thanks!